test/metabase/query_processor_test/aggregation_test.clj

```diff
@@ -53,11 +53,13 @@
 (deftest standard-deviation-test
   (mt/test-drivers (mt/normal-drivers-with-feature :standard-deviation-aggregations)
     (testing "standard deviation aggregations"
-      (is (= {:cols [(qp.test/aggregate-col :stddev :venues :latitude)]
-              :rows [[3.4]]}
-             (qp.test/rows-and-cols
-               (mt/format-rows-by [1.0]
-                 (mt/run-mbql-query venues {:aggregation [[:stddev $latitude]]})))))))
+      (let [query (mt/mbql-query venues {:aggregation [[:stddev $latitude]]})]
+        (mt/with-native-query-testing-context query
```

Comment:
I changed this to use `with-native-query-testing-context` to get the compiled native query in test failures. It helped with working on the Mongo implementation